helix-lsp/src/client.rs

```diff
@@ -1341,6 +1341,56 @@ impl Client {
         Some(self.call::<lsp::request::DocumentSymbolRequest>(params))
     }
 
+    pub fn prepare_file_rename(
+        &self,
+        old_uri: String,
+        new_uri: String,
```

Comment:
these shuold be URLs instead of strings, ints unfortunate that the LSP prototocoal stores these as strings instead of URIs (which they are required to be according to the protocol). You can then easily convert to string below

Reply:
Updated !